.delivery/build-cookbook/recipes/functional.rb

```diff
@@ -31,3 +31,38 @@
     end
   end
 end
+
+# Teardown Acceptance, Union, and Reheasal Omnitruck instances.
+if node['delivery']['change']['stage'] == 'delivered'
+  %w(acceptance union rehearsal).each do |env|
+    machine_batch do
+      # Nodes with name scheme: "omnitruck-env-1"
+      1.upto(instance_quantity) do |i|
+        machine "#{instance_name}-#{i}" do
+          chef_server chef_server_details
+          chef_environment env
+          attribute 'delivery_org', node['delivery']['change']['organization']
```

Comment:
@tduffield I cargo culted this from the `deploy` recipe; can I just drop the attributes since the action is `:destroy`?